src/components/dropdown.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path="../../typings/tsd.d.ts" />
+import {Component, View, NgClass} from 'angular2/angular2';
+
+@Component({
+  selector: 'uib-dropdown',
+  properties: [
+    'open'
+  ]
+})
+@View({
+  directives: [NgClass],
+  template: `<div class="dropdown" [ng-class]="{open: open}">
+    <ng-content></ng-content>
+  </div>`
+})
+export class UibDropdown {
+  private open: boolean;
+}
+
+@Component({
+  selector: 'uib-dropdown-menu',
+  host: {
+    class: 'dropdown-menu'
```

Comment:
TS cries with this for me on the accordion. We will see when I merge the tooling.